sys/docker/Dockerfile

```diff
@@ -42,42 +41,44 @@ RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - \
     && npm rebuild node-sass \
     && rm -rf /var/lib/apt/lists/*
 
-RUN apt-get -y remove \
-        git \
-        gcc \
-        g++ \
-        make \
-    && apt-get -y clean \
-    && apt-get -y autoremove \
-    && rm -rf /var/lib/apt/lists/*
-
-COPY sys/docker/start.sh ./
-
 RUN apt-get update \
     && apt-get install -y --no-install-recommends nginx \
     && mkdir -p /data/nginx/cache \
     && rm -rf /var/lib/apt/lists/*
 
-COPY ./sys/nginx/nginx.conf /etc/nginx/conf.d/default.conf
-
 RUN echo "* * * * *", command: "find /data/nginx/cache/ -type f -newermt '30 seconds ago' | xargs rm" >> /var/spool/cron/crontabs/root \
     && echo "* * * * *", command: "( sleep 30 ; find /data/nginx/cache/ -type f -newermt '30 seconds ago' | xargs rm )" >> /var/spool/cron/crontabs/root
```

Comment:
Could you explain why to run the same this way?

AFAIK crontab has a 1 minute granularity but it is not guarantied to run at the a precise second  .. 
The question is "why we need a half-a-minute" granularity ?

Reply:
moreover .. why to write in var/spool instead of etc?  This way you are bypassing the configuration and could create unattended side effects IMHO

Reply:
Since there's no way to invalidate the cache by using the TTL, unless moving to nginx pro, and since the hit counter is based on the backend increasing the value on redis, this a dirty way of having a "balance" of availability via caching and not losing too many hits. I know there should be a deep refactoring of the logic behind it to address both problems properly.

Reply:
more improvements on this in https://github.com/PUGX/badge-poser/pull/407